docs for vLLM backend (#1802)
lemonade-sdk/lemonade: docs/guide/configuration/vllm.md

# vLLM Backend Options

> Lemonade integrates [vLLM](https://github.com/vllm-project/vllm) as an experimental backend for AMD ROCm GPUs on Linux. vLLM brings three things to Lemonade that the existing backends can't:

Reviewer: ```suggestion
Lemonade integrates [vLLM](https://github.com/vllm-project/vllm) as an experimental backend for AMD ROCm GPUs on Linux. vLLM brings three core benefits:
```

Author: Fixed

1. **Day-0 model support.** vLLM typically supports new transformer architectures within hours of their release on Hugging Face — checkpoints load directly, with no per-architecture porting.

> 2. **True omni-modal support.** First-class support for any-to-any models like Qwen3-Omni that mix text, audio, image, and video across both input and output sides.

Reviewer: Is this actually enabled in Lemonade's vLLM? If not this is misleading.

Author: Rephrased to not mention omni yet

3. **Concurrency and multi-GPU.** Paged-attention KV cache, continuous batching, and chunked prefill scale aggregate throughput with in-flight request count; tensor and pipeline parallelism are supported across multiple GPUs.

> > **Status: experimental.** All vLLM models in the registry carry the `experimental` label and are surfaced only when the experimental opt-in is enabled. The backend has been validated on **gfx1151 (Strix Halo)** and **gfx1150 (Strix Point)**. Prebuilt wheels also exist for `gfx110X` (RDNA3) and `gfx120X` (RDNA4) but those targets have not been exercised end-to-end yet.

Reviewer: There is no documentation on how to do this. I also question whether adding this friction is a good idea?

Reviewer: I just set up vLLM without encountering any "experimental" gate, was this hallucinated?

Author: Removed the experimental tag from the models, but added it to the UI - backend manager.
